package.json

```diff
@@ -1,97 +1,120 @@
 {
-  "name": "cvs-svc-document-retrieval",
-  "version": "1.0.0",
-  "description": "Microservice that retrieves documents",
-  "main": "handler.js",
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/dvsa/cvs-document-retrieval.git"
-  },
+  "name": "cvs-svc-template",
+  "version": "1.1.1",
+  "description": "",
+  "main": "src/index.ts",
```

Comment:
this is probably incorrect and should be `src/handler.ts` since we are distributing it as a lib